lib/popcorn/build.ex

```diff
@@ -298,4 +310,12 @@ defmodule Popcorn.Build do
     File.mkdir!(tmp_dir)
     tmp_dir
   end
+
+  defp to_human_duration(us) do
+    case {us / 1_000_000, us / 1_000, :erlang.float(us)} do
+      {s, _, _} when s > 1.0 -> "#{Float.round(s, 2)}s"
+      {_, ms, _} when ms > 1.0 -> "#{Float.round(ms, 2)}ms"
+      {_, _, us} -> "#{Float.round(us, 2)}us"
+    end
```

Comment:
```suggestion
    us = :erlang.float(us)
    ms = us / 1_000
    s = ms / 1_000
    
    cond do
      s > 1.0 -> "#{Float.round(s, 2)}s"
      ms > 1.0 -> "#{Float.round(ms, 2)}ms"
      true -> "#{Float.round(ms, 2)}ms"
    end
```

Reply:
I've written this way originally but found it less legible. Why do you think it's better?